apps/web/prisma/schema.prisma

```diff
@@ -278,3 +280,23 @@ model Campaign {
 
   team Team @relation(fields: [teamId], references: [id], onDelete: Cascade)
 }
+
+model ApiLog {
+  id          String   @id @default(cuid())
+  teamId      Int
+  method      String
+  path        String
+  status      Int
+  duration    Int // in milliseconds
+  apiKeyId    Int?
+  requestBody Json? // Store request body
+  response    Json? // Store response data
+  error       String? // Store error message if any
+  createdAt   DateTime @default(now())
+
+  team   Team    @relation(fields: [teamId], references: [id])
+  apiKey ApiKey? @relation(fields: [apiKeyId], references: [id])
+
+  @@index([teamId])
```

Comment:
https://www.prisma.io/docs/orm/prisma-schema/data-model/indexes#example-using-sort-and-length-together

should we have a combined index with sort order?

Reply:
you mean like this ?

```diff
- @@index([teamId])
- @@index([createdAt])
+ @@index([teamId, createdAt(sort: Desc)])

```

Reply:
yep